modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala

```diff
@@ -245,6 +245,9 @@ object EpicsUtil {
     os.map(_.setTimeout(t.toMilliseconds.toLong, MILLISECONDS).asRight).getOrElse(SeqexecFailure.Unexpected("Unable to set timeout for EPICS command.").asLeft)
   }
 
+  def safeAttribute[F[_]: Sync, A](get: => CaAttribute[A]): F[Option[A]] =
+    Sync[F].delay(Option(get).flatMap(x => Option(x.value)))
```

Comment:
I had a reason to not put the `get` in an `Option`. The functions `getXXX` use a `String` name to find the attribute that was created from the xml files. If it fail, it is because there was an error in the channel definition, most likely a typo in the channel name. If that happens, I wanted to know it immediately.  That is an error with the code, not an execution error, and protecting the `get` with an `Option` would make it harder to find.

Reply:
I understand. I think this would be better handled at compile time but that is a bigger change. I'll revert it to `.get.value`

Reply:
I see you took care of this while I was reviewing 👍

Reply:
I agree that the best scenario would be to have it handled at compile time. We could write a tool that the names used in the code exist in the xml file.